README.md

```diff
@@ -7,7 +7,7 @@ have extended and released under the same [`LICENSE`](./LICENSE)
 Installation
 ------------
 
-**The current version of this library requires gRPC version 1.45.0. Newer versions may work but have not been tested.**
+**The current version of this library requires gRPC version 1.64.0. Newer versions may work but have not been tested.**
```

Comment:
This is not true.

Reply:
Sorry forgot to revert this note. What about:
```suggestion
**The current version of this library requires gRPC version 1.45.0 and was also tested with 1.67.0. Other versions may work but have not been tested.**
```